crates/electrum/src/electrum_ext.rs

```diff
@@ -165,6 +136,7 @@ pub trait ElectrumExt<A> {
     /// The scan for each keychain stops after a gap of `stop_gap` script pubkeys with no associated
     /// transactions. `batch_size` specifies the max number of script pubkeys to request for in a
     /// single batch request.
+    #[allow(clippy::type_complexity)]
```

Comment:
Hmm I don't think we should do this. Can we just package these tuple types into their own types. Consider bundling (`local_chain::Update, RelevantTxids)` into their own types.